src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -161,6 +161,12 @@ export class CDMaterial extends Component<CDMaterialProps> {
     })
   }
 
+  deployTrigger(e) {
+    e.stopPropagation()
+    this.props.triggerDeploy(this.props.stageType)
+    // this.props.redirectToCD()
```

Comment:
Is this expected? If not used remove the commented code.

Reply:
We are not redirecting to app details when user hits deploy as per discussion with Utkarsh. In future we will use this, hence commented for now